### DISCUSSION TITLE
v7.3.3 Release of PowerShell

### DISCUSSION THREAD

**Discussion by SeeminglyScience** [PowerShell/PowerShell#19209] [General]
### Build and Packaging Improvements

<details>

<summary>

<p>Bump to use .NET 7.0.3</p>

</summary>

<ul>
<li>Update third party notices for v7.3.3 (Internal 24353)</li>
<li>Add tool to trigger license information gathering for NuGet modules (#18827)</li>
<li>Update global.json to 7.0.200 for v7.3.3 (Internal 24334)</li>
<li>Update cgmanifest for v7.3.3 (Internal 24338)</li>
</ul>

</details>

[7.3.3]: https://github.com/PowerShell/PowerShell/compare/v7.3.2...v7.3.3

### SHA256 Hashes of the release artifacts

- hashes.sha256
  - FC79F58BC2864298E87B2CD742120A5F5593E53EACE0A09C7432A75763BB4A40
- powershell_7.3.3-1.deb_amd64.deb
  - 40BEE3DEB9AC71DFD86A7DBF2F2BDCB8D039FA454DC9D705D579E4957194FCD4
- powershell-7.3.3-1.cm.x86_64.rpm
  - 1CB618E66676CA2383DC0C6DA03B1FE66DBBBF17C1CCFFB0CD91634812064F07
- powershell-7.3.3-1.rh.x86_64.rpm
  - 0B2C0B0D4D042C20A88AA7F4D14F9A83005B596B4D1C15877C2B268F2F777194
- powershell-7.3.3-linux-alpine-x64.tar.gz
  - 4BC39FB7A0F486AB498C6DF2AD599DB79AC2B56DF768485D48541639DDE3A170
- powershell-7.3.3-linux-arm32.tar.gz
  - 8F10ADE3BD12A21FBA51BCAAECB12B79CB11092725AC0F7C7E6947C053053766
- powershell-7.3.3-linux-arm64.tar.gz
  - 5EFCA750F22BFECB31BCDA57C3A76BB804C68AEF6626F123AAC68C6E3D7E52F7
- powershell-7.3.3-linux-x64.tar.gz
  - 478320F62C9BEC5765414B82B4086310777BF95AB2FC2310B2DB6CBC253B40B4
- powershell-7.3.3-linux-x64-fxdependent.tar.gz
  - 90904B81A7EA7DFDA7DBE971833DBD50DD925965C583D492114A5F0709F3A9C8
- powershell-7.3.3-osx-arm64.pkg
  - BB72A99292C708CEFEF78B2BDA56C49BAED5469A00F29B174D921E88C32BF457
- powershell-7.3.3-osx-arm64.tar.gz
  - F9717075A85F5B631ED9EC3C68C2AEB91F1FEF2AC18B62F26FD7F2BF57F5D876
- powershell-7.3.3-osx-x64.pkg
  - 476DE8467E6C68F32F798E3212F9C48E8B7D108F86029C3BE82BFE2C19223BB4
- powershell-7.3.3-osx-x64.tar.gz
  - 80D354E1DFE6B28ADF9BCF41407810CE9160EEF30CE6732754EAF497576E0D27
- PowerShell-7.3.3-win-arm32.zip
  - 2D5152B97FBBC22A5AF7FB4455A643B9867559BB27DB275423A60C133577DF9E
- PowerShell-7.3.3-win-arm64.zip
  - 1F382C91C94E31B9E04FBBDC63ACB7048869B30E8BDAFA7C1D365A1E7D130279
- PowerShell-7.3.3-win-fxdependent.zip
  - B3C47105A9BE5748A13754FD86EA491FD84EB438D314D74170DE811D8F1680CA
- PowerShell-7.3.3-win-fxdependentWinDesktop.zip
  - 7F042D16E1C05D169840AE1BDD12F020C9CC6AEFEF52F33D529361522B830235
- PowerShell-7.3.3-win-x64.msi
  - E3C58761D9D0863CB7EE23170108EB47B675DA7B0FFEE3412D3630F7B249452E
- PowerShell-7.3.3-win-x64.zip
  - 4B8F76445EF2D74BB0A747B0EA4F7E54A1E68FDDCEF0DE140E938028C9DE0C8C
- PowerShell-7.3.3-win-x86.msi
  - E3A879E59624DF732893C7582CB915A44F67C28B765F81EE9269D2D8D998C9E5
- PowerShell-7.3.3-win-x86.zip
  - C0DE53EFAA3A80E928F79C90169CED8315B0FB7AF942295AC19641039DDA5E25

<hr /><em>This discussion was created from the release <a href='https://github.com/PowerShell/PowerShell/releases/tag/v7.3.3'>v7.3.3 Release of PowerShell</a>.</em>

**Comment by Norpan70** [comment]
Is there a command to update from PowerShell in Windows 11? It's pretty cumbersome to download an install via web browser.

**Comment by corn-collns** [comment]
An authentication username and password is required
